trivy-image-scan/action.yml

```diff
@@ -59,13 +59,15 @@ runs:
       shell: bash
       run: |
         echo "FORMAT=json" >> $GITHUB_ENV
-        echo "OUTPUT=''trivy-results.json''" >> $GITHUB_ENV
+        echo "OUTPUT=trivy-results.json" >> $GITHUB_ENV
+
 
     - name: Set up output values
       if: ${{ inputs.output-mode == 'log' }}
       shell: bash
       run: |
         echo "FORMAT=table" >> $GITHUB_ENV
+        echo "OUTPUT=" >> $GITHUB_ENV
```

Comment:
when i run twice with different output method for ex, json and log formats.  In the first run, it will have output='results.json' and this is also carried in the next run in the same job. So, the output of the log gets stored in the results.json which is set in the previous run. This changes will initialize the output path to null thus outputs to the console.

Reply:
Got it. These really shouldn't have ever been exported since they're for internal use. Rather than unsetting it on the way in, could we add an unconditional final step to clean up and remove the variables? It also would make sense to give them more unique names  (e.g. TRIVY_FORMAT) to avoid conflicts with any existing variables.